CartManagement/cart.css

```diff
@@ -0,0 +1,205 @@
+@import url("https://infusify.netlify.app/styles/Infusify.css");
+
+:root{
+    --blue-hover:rgb(22, 84, 134);
+    --pink-hover:rgb(216, 144, 216);
+    --hot-pink-hover:rgb(95, 15, 66);
+}
+
+.flex-container{
+    display:flex;
+}
+.flex-rows{
```

Comment:
No need to specify the class for row it is default behaviour of grid

Reply:
Okay will change that